js/search-tabs-carousel.js

```diff
@@ -0,0 +1,146 @@
+let courses = {
+    "Python": {},
+    "Excel": {},
+    "Web-Development": {},
+    "JavaScript": {},
+    "Data-Science": {},
+    "AWS-Certification": {},
+    "Drawing": {}
+};
+
+let activeTab = document.querySelector('#python');
+let activeTabHeader = document.querySelector('#tab-header');
+let activeTabDescription = document.querySelector('#tab-description');
+let activeTabButton = document.querySelector('#tab-button');
+
+const getCourses = async () => {
+    let result = await fetch(`https://api.jsonbin.io/v3/b/63065ce65c146d63ca7dbcbc`);
+    let myJSON = await result.json();
+    for (let track in courses) {
+         courses[track] = myJSON['record'][track];
+    }
+
+    let key = 'Python';
+    activeTabHeader.innerText = courses[key]['header'];
+    activeTabDescription.innerText = courses[key]['description'];
+    activeTabButton.innerText = `Explore ${key}`;
+    buildCourses(key);
+}
+window.addEventListener('DOMContentLoaded', () => getCourses());
+
+const form = document.querySelector('form');
+form.addEventListener('submit', function (event) {
+    event.preventDefault();
+
+    let key = activeTab.innerText.replace(' ', '-');
+    buildCourses(key);
+});
+
+let coursesNav = document.querySelector('#courses-tabs');
+coursesNav.addEventListener('click', function (event) {
+    event.preventDefault();
+    //box.value = '';
+
+    let newActiveTab = event.target;
+    if (activeTab != newActiveTab) {
+        activeTab.classList.remove('active');
+        newActiveTab.classList.add('active');
+
+        let key = event.target.innerText.replace(' ', '-');
```

Comment:
Extract this line into another function so we can understand what it does, for example createCategoryFetchKey